app_template_iframe/assets/iframe.html

```diff
@@ -2,7 +2,7 @@
 <head>
   <meta charset="utf-8">
   <!-- http://garden.zendesk.com -->
-  <link rel="stylesheet" href="https://assets.zendesk.com/apps/sdk-assets/css/2/zendesk_garden.css" type="text/css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@zendeskgarden/css-bedrock@7.0" type="text/css">
```

Comment:
Could this have been an alias to `latest` instead (jsdelivr supports something like that, yea)?

Reply:
Maybe? If developers leave `latest` in could it eventually cause breaking changes?